LongestPalindromicSubstring.py

```diff
@@ -0,0 +1,21 @@
+class Solution:
+    def longestPalindrome(self, s: str) -> str:
+        L = R = l = r = 0
```

Comment:
Naming is hard 😢 Wouldn't be better to name L, R as `resultLeft, resultRight`?

Reply:
You are right, sorry